shared/src/ClientPropertyCollection.cs

```diff
@@ -73,44 +97,42 @@ public void Add(IDictionary<string, object> properties, string componentName = d
         /// <param name="version"></param>
         /// <param name="description"></param>
         /// <param name="componentName"></param>
-        public void Add(string propertyName, object propertyValue, int statusCode, long version, string description = default, string componentName = default)
+        public void Add(string componentName, string propertyName, object propertyValue, int statusCode, long version, string description = default)
         {
             if (Convention?.PayloadSerializer == null)
```

Comment:
q = would this ever be null, we'd always default this to newtonsoft, right?

Reply:
actually nvm, this is right -> null when created from scratch and value available if reusing from a returned property.